gulpfile.js

```diff
@@ -106,39 +112,79 @@ gulp.task('translations-feelup', ['prepare-translations'], function () {
                     item.msgstr = [''];
                 }
                 if (0 == item.msgstr[0].length) {
-//                    item.msgid[0] = item.msgid;
+                    // item.msgid[0] = item.msgid;
                     item.msgstr[0] = item.msgid;
                 }
                 return item;
             }
         }))
         .pipe(gulp.dest(languagesFolder));
-});
+}
 
 // Cleanup temporary translation folders.
-gulp.task('cleanup', ['prepare-translations'], function () {
-    var folders = getFolders(languagesFolder);
-
+function cleanup() {
+    prepare_translations();
     return folders.map(function (folder) {
         return gulp.src(path.join(languagesFolder, folder), {read: false})
             .pipe(clean());
     });
-});
+}
 
 // Compile *.po to *.mo binaries for usage.
-gulp.task('compile-translations', ['translations-feelup'], function () {
+function compile_translations() {
+    translations_feelup();
     // Compile POs to MOs.
     return gulp.src(languagesFolder + '*.po')
         .pipe(gettext())
         .pipe(gulp.dest(languagesFolder))
-});
-
-gulp.task('default', [], function () {
-    gulp.run('prepare-source');
-    gulp.run('update-transifex');
-    gulp.run('download-translations');
-    gulp.run('prepare-translations');
-    gulp.run('translations-feelup');
-    gulp.run('cleanup');
-    gulp.run('compile-translations');
-});
\ No newline at end of file
+}
+
+// Run postcss processing for styles.
+function style() {
+    let plugins = [
+        cssnano()
+    ];
+
+    return gulp.src( './assets/scss/**/*.scss' )
+        .pipe( sass() )
+        .pipe( postcss( plugins ) )
+        .pipe( gulp.dest( './assets/css/' ) )
+}
+
+// Compile css only in dev mode.
+function watch() {
+    gulp.watch( './assets/scss/*.scss', style );
```

Comment:
@bahiirwa You can use the blob `./assets/scss/**/*.scss` instead of the two `watch`.